src/class-supported-post-types.php

```diff
@@ -93,4 +93,13 @@ public function register_post_meta(): void {
 			]
 		);
 	}
+
+	/**
+	 * Should we register the block in the current context.
+	 *
+	 * @return boolean
+	 */
+	public function should_register_block() {
```

Comment:
Should this be applied here too?

https://github.com/alleyinteractive/wp-curate/blob/ce2e532d6d00be987e521e29b02ce1796b9a85d7/entries/slotfills/index.php#L22-L27